libs/application/templates/no-debt-certificate/src/fields/ConfirmationPendingField/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react'
+import { useMutation } from '@apollo/client'
+import { FieldBaseProps, DefaultEvents } from '@island.is/application/core'
+import { Box, LoadingDots } from '@island.is/island-ui/core'
+import { SUBMIT_APPLICATION } from '@island.is/application/graphql'
+
+export const ConfirmationPendingField: FC<FieldBaseProps> = ({
+  application,
+  refetch,
+}) => {
+  const applicationId = application.id
+
+  const [finishSubmit, setFinishSubmit] = useState<boolean>(false)
+
+  const [submitApplication] = useMutation(SUBMIT_APPLICATION, {
+    onError: (e) => console.error(e.message),
```

Comment:
Shouldn't this use the `handleServerError` utility in `application/ui-components`?